msb/network/zmq/publisher.py

```diff
@@ -14,7 +15,7 @@ def __init__(self, config: ZMQConf):
         self.context = zmq.Context.instance()
         self.socket = self.context.socket(zmq.PUB)
 
-        self.packer = json
+        self.packer = get_packer(config.packstyle)
```

Comment:
I think this is named `packer` now in the config. I don't have a clear preference whether we should use packer or packstyle.

Reply:
i think this should be named packstyle and unpackstyle respectivel